.github/workflows/ci.yml

```
@@ -1,39 +1,113 @@
       run: cmake -P cmake/spell.cmake
 
   coverage:
-    needs: [lint]
-
+    needs: [ lint ]
     runs-on: ubuntu-24.04
 
     container:
-      image: jbeirer/alma9-fastcalosim:latest
-
+      image: fcsproj/fastcalosim-alma9:${{ env.CONTAINER_TAG }}
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)